Automattic/wp-calypso
# Help Center: add postcss-prefix-selector

Bug fix for p1723148019985739-slack-C02FMH4G8

## Proposed Changes

Add postcss prefixer to prefix `.help-center` to serach component style.

## Why are these changes being made?

Search component is bleeding styles into frontend.

## Testing Instructions

Pull branch
yarn dev --sync from apps/help-center
Sandbox widgets.wp.com and make sure Help Center search works
Test on the frontend of Atomic site as well with search block
